requirements/docs.txt

```diff
@@ -1,6 +1,4 @@
-# Sphinx 1.7.8 has a bug when building on cached envs
-# https://github.com/sphinx-doc/sphinx/issues/5361
-sphinx>=1.3,!=1.7.8
+sphinx>=1.8
```

Comment:
If we apply the 24-month rule (somewhat proposed in https://github.com/scikit-image/scikit-image/issues/4100#issuecomment-522331398), we should go with 1.7.4 (release date Apr 25, 2018). 1.8.0 was released on Sep 12th, 2018.

I'm personally fine with having 1.8.0, since this a pure doc dependency, and we can be more agressive with them. Just wanted to make sure that we are on the same page regarding those policies :slightly_smiling_face:.

Reply:
There is also @emmanuelle practical argument, which is: are we going to prevent people from using scikit-image if we increase the min version? (See https://github.com/scikit-image/scikit-image/pull/4568#issuecomment-611654645). In this case likely not. :)

Also, if we decide to set 1.7.4, we need to revert and hold on the plugin. Here, I loop back on the fact that we must extensively test our library against all the min versions.

Reply:
However, you raise an important point @soupault we probably need to think about a more solid policy on this, especially for scikit-image 1.0.

Reply:
I feel I want to write a SKIP on this topic! Give me a day.

Reply:
@soupault remember that SKIPs are for controversial proposals. =P Start with an RFC, and if everyone agrees, we can adopt the policy. =)

Reply:
@jni, @soupault is working by anticipation :laughing: